module/API/Measurer.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * @file Measurer.h
+ * @brief 計測に用いる関数をまとめたクラス
```

Comment:
.cppと.hの@briefは揃えましょう
```suggestion
 * @brief 計測に用いる関数をまとめたラッパークラス
```